pkgs/applications/science/meteorology/supercell-wx/default.nix

```diff
@@ -0,0 +1,154 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  aws-sdk-cpp,
+  bzip2,
+  cmake,
+  ninja,
+  zlib,
+  openssl,
+  curl,
+  glew,
+  geos,
+  git,
+  boost185,
+  spdlog,
+  stb,
+  libcpr,
+  libpng,
+  libSM,
+  geographiclib,
+  howard-hinnant-date,
+  re2,
+  gtest,
+  glm,
+  qtbase,
+  qttools,
+  qtmultimedia,
+  qtpositioning,
+  qtimageformats,
+  tbb_2021_11,
+  tzdata,
+  substituteAll,
+  python3,
+  wrapQtAppsHook,
+}:
+let
+  version = "0.4.4";
+  src = fetchFromGitHub {
+    owner = "dpaulat";
+    repo = "supercell-wx";
+    rev = "refs/tags/v${version}-release";
+    sha256 = "sha256-HghyRFLKboztQEOsvN2VaSxxcuPPu1GIakBNWRIWBOk=";
+    fetchSubmodules = true;
+  };
+
+  gtestSkip = [
+    # Skip tests requiring network access
+    "AwsLevel*DataProvider.FindKeyNow"
+    "AwsLevel*DataProvider.FindKeyFixed"
+    "AwsLevel*DataProvider.LoadObjectByKey"
+    "AwsLevel*DataProvider.Refresh"
+    "AwsLevel*DataProvider.GetAvailableProducts"
+    "AwsLevel*DataProvider.GetTimePointsByDate"
+    "AwsLevel*DataProvider.Prune"
+    "UpdateManagerTest.CheckForUpdates"
+    "WarningsProvider*\"https"
+
+    # These tests are failing (seemingly can't overwrite a file created by earlier test).
+    "SettingsManager/DefaultSettingsTest*"
+    "SettingsManager/BadSettingsTest*"
+  ];
+in
+stdenv.mkDerivation {
+  name = "supercell-wx";
+  inherit version;
+  inherit src;
+
+  meta = {
+    homepage = "https://supercell-wx.rtfd.io";
+    downloadPage = "https://github.com/dpaulat/supercell-wx/releases";
+    description = ''
```

Comment:
Thanks for the input! I will work to address these.

Reply:
@FliegendeWurst, I have:

- Rebased onto latest and squashed the package commits
- Moved the original description to `longDescription` and added a shorter `description`.
- Fixed the build errors. I'm not completely happy with part of the solution (add more `-Wno-error` flags) but I  think they will need to be addressed upstream. There is a new version available, `0.4.5`, that includes one of my patches but it looks like it will be another small project to update to it.